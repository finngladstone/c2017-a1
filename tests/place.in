who
who
place A1
who
place A1
place A2
who
place A2
history
place !@#$%^&*()
place @#$%^& #$%^&*
place A12 222222222222222222222
place AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAa
place AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA A12
place A12 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
history
place A20
place B20
who
place A19
who
place A19
place X20
place Y70
history
lalal 40
lalala 4040440 40 40 40
i will use my C experience to sell cheats and make an exorbiant sum :D
term
