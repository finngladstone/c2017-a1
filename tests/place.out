B
B
W
Occupied coordinate
B
Occupied coordinate
A1A2
Invalid coordinate
Invalid!
Invalid!
Invalid coordinate
Invalid!
Invalid!
A1A2
Invalid coordinate
Invalid coordinate
B
W
Occupied coordinate
Invalid coordinate
Invalid coordinate
A1A2A19
Invalid!
Invalid!
Invalid!
